docs/topics/genericwebhook.md

```diff
@@ -0,0 +1,45 @@
+# Generic Webhook
+
+Brigade contains a Generic Webhook which is part of its API Server and can be used to accept requests from other platforms or systems.
+
+Generic Webhook is _not enabled by default_.
+
+## Intro to Generic Webhook 
+
+Brigade API Server can optionally be activated to accept `POST` webhook requests at `/webhook/:projectID/:secret` path. When this endpoint is called, Brigade will respond by creating a Build with a `webhook` event. This provides Brigade developers with the ability to trigger scripts based on messages received from any platform that can send a POST HTTP request.
+
+## Configuring Brigade API Server for Generic Webhook
+
+Generic Webhook support is disabled by default, but can easily be turned on during installation or upgrade of Brigade:
+
+```
+$ helm install -n brigade brigade/brigade --set generic-gateway.enabled=true
```

Comment:
According to the chart, this should be `--set genericGataway.enabled=true` - https://github.com/Azure/brigade-charts/blob/f03f5a12a8df3eba12fdb44c4ac75282dbcbf7a5/charts/brigade/values.yaml#L49